src/esp/sensor/VisualSensor.h

```diff
@@ -18,12 +19,57 @@ class RenderTarget;
 }
 
 namespace sensor {
-
+struct VisualSensorSpec : public SensorSpec {
+  VisualSensorSpec();
+  SensorType sensorType = SensorType::Color;
+  float near = 0.01f;
+  float far = 1000.f;
+  Mn::Deg hfov = Mn::Deg{90.f};
+  float ortho_scale = 0.1f;
+  vec2i resolution = {84, 84};
+  std::string encoding = "rgba_uint8";  // For rendering colors in images
+  bool gpu2gpuTransfer = false;         // True for pytorch tensor support
+  void sanityCheck() {
```

Comment:
Sanitycheck also need to check the value. For example, width or height cannot be <=0.

Reply:
Thanks for the guidance!